boards/shields/seeed_xiao_round_display/seeed_xiao_round_display.overlay

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2024, Nicolas Goualard <nicolas.goualard@sfr.fr>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+    chosen {
+        zephyr,display = &gc9a01_xiao_round_display;
+    };
+
+    vbatt {
+        compatible = "voltage-divider";
+        io-channels = <&adc 0>;
+        output-ohms = <470000>;
+        full-ohms = <940000>;
+    };
+
+    lvgl_pointer {
+        compatible = "zephyr,lvgl-pointer-input";
+        input = <&chsc6x_xiao_round_display>;
+    };
+
+    aliases {
+        rtc = &pcf8563_xiao_round_display;
+    };
+};
+
+&xiao_serial {
+    status = "disabled";
+};
```

Comment:
would be nice to have a comment to explain why it's there (or maybe it's not needed and should be dropped?)

Reply:
The pins are used by the display and touch controller. I added a comment explaining that in the overlay.